README.org

```diff
@@ -1,977 +1,3 @@
-         because of lenovo's implementation, you need to apply a patch fix
-         either way
-         - it involves patching the kernal or editing boot configs
-         - didn't mention them earlier because this is an idiot's guide and i
-           didn't want to overwhelm you
-           - i was also overwhelmed, i was the idiot
-       - reference the wiki, make your decision:
-         - if you want to return to S3, follow this guide: https://delta-xi.net/#056
-         - it is very good and really does guide you through all the steps, do
-           not be intimidated
-    2. if you follow the s3 guide (as I did), some advice:
-       1. *first and foremost, update your bios as described above*
-          1. good chance your s3 won't work nearly as good without it (if at
-             all, going by forum discussions)
-       2. when running the patch, Hunk 7 failed for me.
-          1. solution: there is a function call that will look like this in
-             your .dsl:
-             #+BEGIN_SRC asm
-               OperationRegion (GNVS, SystemMemory, 0x4FF4E000,0x0771)
-             #+END_SRC
-             - the last two values are different from the ~.patch~ the author
-               provides. Edit the values *in the patch* to be the *same* as
-               those in your *.dsl*
-             - after this, *regenerate* the .dsl as the author guides you and
-               reapply the patch as instructed, hunk 7 should now pass.
-          2. the author uses ~GRUB~, if you followed my guide, you don't use
-             ~GRUB~ like me.
-             1. Follow the guide as instructed up to, and including, to where
-                we ~cp~ the ~acpi_override~ to ~/boot/~
-             2. Now we need to edit our ~arch.conf~ entry that we made earlier
-                in ~/boot/loader/entries/~
-                - we are making two additions, adding a ~initrd /acpi_override~
-                  entry and then adding another kernal param argument to
-                  options, ~mem_sleep_default=deep~
-                - this is what my ~arch.conf~ looks like:
-                  #+BEGIN_SRC conf
-                    title Arch Linux
-                    linux /vmlinuz-linux
-                    initrd /intel-ucode.img
-                    initrd /acpi_override
-                    initrd /initramfs-linux.img
-                    options cryptdevice=UUID=<LONG-ALPHANUM-STRING>:cryptlvm root=/dev/mapper/main_group-root mem_sleep_default=deep quiet rw
-                  #+END_SRC
-                  - notice the fourth line and the end of the last line, you'll
-                    see my edits as compared to when we first made this file
-                  - I've excluded the edit for enabling hibernation from this
-                    example, fyi
-*** hibernate support
-    - changes to HOOKS
-    - regenerating initramfs
-    - adding param to arch.conf loader file in options 
+  [[https://www.ubuntu.com/download/desktop][Don't.]]
```

Comment:
wrong link, i think you meant to use [this](https://www.microsoft.com/en-us/software-download/windows10ISO)